### ISSUE TITLE
Accuracy() function

### ISSUE THREAD

**Issue by Assiam** [robjhyndman/forecast#105]
Hi!

I'm trying to compute the accuracy for my forecasting but I found that the accuracy change when I convert my test from time-series to numerical vector especially for the measure : MASE . I would like to understand why and what is the correct output?

Example :

> Dem2=window(y2,end=c(2011,12))
> Test<-window(y2,start=c(2012,1))
> fit<-ets(Dem2)
> f1<-forecast(fit,12)
> round(accuracy(f1,Test),2)
                ME RMSE  MAE   MPE MAPE MASE ACF1 Theil's U
Training set -0.01 0.07 0.05 -0.05 0.27 0.08 0.27        NA
Test set     -0.08 0.14 0.12 -0.47 0.65 0.17 0.58      1.06

> Test=as.vector(Test)
> Test
 [1] 18.72528 18.80953 18.64780 18.65369 18.51886 18.59252 18.33444 18.36091 18.14653 17.96715
[11] 18.01493 17.92674
> round(accuracy(f1,Test),2)
                ME RMSE  MAE   MPE MAPE MASE ACF1
Training set -0.01 0.07 0.05 -0.05 0.27  0.5 0.27
Test set     -0.08 0.14 0.12 -0.47 0.65  1.1   NA

Thank you for your help and for your work

**Comment by robjhyndman** [comment]
Read the help file. You lose the seasonal information by converting to a numerical vector.
